library/quality/quality.gradle

```diff
@@ -77,6 +77,7 @@ android {
                 'IconExpectedSize',
                 'InvalidPackage', // Firestore uses GRPC which makes lint mad
                 'NewerVersionAvailable', 'GradleDependency' // For reproducible builds
+                'MissingTranslation' // TODO remove in future PR
```

Comment:
I think you're also going to have to add `UnusedResources`. And you need a comma above. 😄

Reply:
Done, thanks!